$NetBSD: patch-alr__env.gpr,v 1.4 2026/06/06 06:20:24 dkazankov Exp $

Add NetBSD support
Change build type to avoid runtime check exceptions

--- alr_env.gpr.orig	2026-05-25 15:21:25.000000000 +0300
+++ alr_env.gpr
@@ -40,6 +40,7 @@
    --  Set environment variables for dependencies
 
    case Alire_Common.Host_Os is
+      when "netbsd" => for External ("GNATCOLL_OS") use "unix";
       when "freebsd" => for External ("GNATCOLL_OS") use "unix";
       when "openbsd" => for External ("GNATCOLL_OS") use "unix";
       when "linux"   => for External ("GNATCOLL_OS") use "unix";
@@ -48,7 +49,7 @@
    end case;
 
    for External ("LIBRARY_TYPE") use "static";
-   for External ("BUILD") use "DEBUG";
+   for External ("BUILD") use "PROD";
 
    --  The user must set ALIRE_OS before building. See the list of valid values
    --  in alire_common.gpr
